### DISCUSSION TITLE
Enable post upgrade script

### DISCUSSION THREAD

**Discussion by ACiDGRiM** [nextcloud/all-in-one#5791] [Questions] [answered]
I've found that every time I upgrade I need to manually run app updates, and re-enable apps, specifically MFA providers such as totp. This means that I need to connect into the container and run a fixup.sh script that completes this.

It would be helpful if I could provide the path of a post upgrade custom script in env that would run only after a sucessful upgrade.

**Comment by szaimen** [comment, marked as answer]
Hi, before going this route, can you post the nextcloud container logs here after an update fails like this? (It is not the intended behaviour and rather a bug if this happens)...
